explore-cardano/understanding-the-delegation-mechanism.md

```diff
@@ -1,5 +1,34 @@
 # Understanding the Delegation Mechanism
 
-As Cardano is a proof-of-stake system, owning ada not only allows you to buy goods or services, but also confers upon you the right and obligation to participate in the protocol and create blocks. 
+The stake delegation is a mechanism added to the Cardano proof of stake (PoS) protocol that allows the protocol to scale even in a setting where the set of stakeholders is highly fragmented.
 
-These two uses can be separated by the delegation mechanism, meaning someone who owns ada can keep the spending power, while delegating the power to participate in the protocol to someone else, a stake pool. It is important to note that funds can be spent normally at any time, regardless of how they are delegated.
+Anyone who owns ada can participate in the stake delegation process while retaining their spending power, while simultaneously delegating the power to participate in the protocol to a stake pool. Note that you can spend your ada normally at any time, regardless of how you delegated it. This mechanism will enable stakeholders to delegate their “voting rights” i.e., the right to participate in the slot leader election  process in each epoch.
+
+Stake delegation gives rise to “stake pools” that act in the similar way to mining pools in the Bitcoin protocol. Stake pool operators must be online in order to generate blocks when they are selected as slot leaders.
+
+## Stake delegation requirements
+
+Delegating stake requires posting two certificates to the chain: a staking key registration, and a delegation certificate. Posting certificates requires funds, so a user setting up their first wallet will need a bootstrapping mechanism. This mechanism relies on the possibility of base addresses using a staking key before posting the registration certificate for that key.
```

Comment:
It's actually a stake address, not just a staking key. A stake address can be based either on a single key (the normal case) or on a script (multi-sig now, or later Plutus).